xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
+     By default, Linux offers 6 different consoles, that can be switched to
+     via the key-combinations
+     <keycombo><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="alt"/><keycap>F6</keycap></keycombo>, when
+     running in text mode, or
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F6</keycap></keycombo>
+      when running in a graphical session. The
+      associated terminal devices are named <literal>tty1</literal> .. <literal>tty6</literal> accordingly.
+    </para>
+
+    <para>
+     The following steps restrict root access to the first TTY. Even this
+     access method is only meant for emergency access to the system and should
+     never be used for everyday system administration tasks.
+    </para>
+
+    <itemizedlist>
+     <listitem>
+     <para>
+      Ensure that the PAM stack configuration file <filename>/etc/pam.d/login</filename>
+      contains the <literal>pam_securetty</literal> module in the <literal>auth</literal> block:
+     </para>
+     <screen>auth     requisite      pam_nologin.so
+auth     [user_unknown=ignore success=ok ignore=ignore auth_err=die default=bad] pam_securetty.so noconsole
+auth     include        common-auth</screen>
+     </listitem>
+     <listitem>
+      <para>
+       This will include the <literal>pam_securetty</literal> PAM module during the
+       authentication process on local consoles to allow root only to login on
+       tty devices that are listed in the file <filename>/etc/securetty</filename>.
+       Thus to actually limit the tty devices for root you need to remove all
+       entries from this file except one:
+      </para>
+     <screen>#
+# This file contains the device names of tty lines (one per line,
+# without leading /dev/) on which root is allowed to login.
+#
+tty1</screen>
+     </listitem>
+     <listitem>
+      <para>
+       After these changes root logins from non-allowed terminals will be
+       rejected right away without even querying the root password.
+      </para>
+     </listitem>
+    </itemizedlist>
+
+    <important>
+     <para>
+      One might be tempted to add the <literal>pam_securetty</literal> PAM
+      module to the <filename>/etc/pam.d/common-auth</filename> file to make it
+      effective for all PAM services. This would break <literal>su</literal>
+      and <literal>sudo</literal>, however, because these tools would then
+      also reject root authentications.
+     </para>
+    </important>
+
+    <important>
+     <para>
+      These configuration changes will also cause root logins on serial
+      consoles like <filename>/dev/ttyS0</filename> to be denied. In case you
+      require such use cases you need to list the respective tty devices
+      additionally in the <filename>/etc/securetty</filename> file.
+     </para>
+    </important>
+
+    <note>
```

Comment:
I think, this note is essential for the procedure above. Maybe move that *before* the procedure?

Reply:
fine with me, I'll move if up